Automattic/Gridicons-iOS
# Add new Gridicons

This PR adds a load of new Gridicons that have been added to the main repo since Gridicons-iOS was last updated:

* chat
* coupon
* crossCircle
* ellipsisCircle
* filter
* fullscreen
* fullscreenExit
* money
* offline
* pause
* product
* productDownloadable
* productExternal
* productVirtual
* readMore
* refund
* resize
* shipping
* userAdd

Because the code is auto-generated, the code for _all_ of the Gridicons will have been updated.

**To test:**

* Run the Gridicons demo and look through all the icons to check everything looks okay.
* Reference this branch from WPiOS's Podfile, and check everything looks okay in the main app.

Needs review: @jleandroperez
